content/articles/zone-verification.md

```diff
@@ -16,7 +16,7 @@ categories:
 
 ## What is Zone Verification?
 
-Zone Verification is a security measure designed to ensure that only the current owner of a domain can add and manage its DNS zone in DNSimple, especially when the domain was previously associated with another DNSimple account.
+Zone Verification is a security measure designed to ensure that a DNS zone can only be added to DNSimple with the domain owner's (maybe registrant?) permission, especially when the domain was previously associated with another DNSimple account and still points at DNSimple.
```

Comment:
I think we can remove the question mark that I suggested. 😅 🙈

Reply:
I did, it isn't in the final copy.

Reply:
Neat-o! Thanks!